sentinel-core/src/main/java/com/alibaba/csp/sentinel/slots/block/flow/FlowRuleManager.java

```diff
@@ -55,16 +55,26 @@
     private static final FlowPropertyListener LISTENER = new FlowPropertyListener();
     private static SentinelProperty<List<FlowRule>> currentProperty = new DynamicSentinelProperty<List<FlowRule>>();
 
+    /** the corePool size of SCHEDULER must be set at 1, so the two task ({@link #startMetricTimerListener()}, {@link #startMetricBeanTimerListener()}) can run orderly by the SCHEDULER **/
     @SuppressWarnings("PMD.ThreadPoolCreationRule")
     private static final ScheduledExecutorService SCHEDULER = Executors.newScheduledThreadPool(1,
         new NamedThreadFactory("sentinel-metrics-record-task", true));
 
     static {
         flowRules.set(Collections.<String, List<FlowRule>>emptyMap());
         currentProperty.addListener(LISTENER);
+        startMetricBeanTimerListener();
         startMetricTimerListener();
     }
     
+    /**
+     * start the MetricBeanTimerListener at the fix rate, the initialDelay is different with the {@link #startMetricTimerListener()},
+     * so that the two task will start running at different time.
+     */
+    private static void startMetricBeanTimerListener() {
+        SCHEDULER.scheduleAtFixedRate(new MetricBeanTimerListener(), 0, 1, TimeUnit.SECONDS);
+    }
+    
```

Comment:
I found that we can config the interval with `csp.sentinel.metric.flush.interval` in `startMetricTimerListener`,
but `startMetricBeanTimerListener` has one second fixed interval. Do we need to make it configurable? At present, this function is enabled by default, maybe some users want to disable it.

Reply:
yes, that's a good idea! I will change it later.